book/hardware-battery.md

```diff
@@ -1,38 +1,27 @@
 # Battery Interface
 
-## Overview
-
 Solo's battery is used to power Solo, and needs regular recharging. This document specs out the interface to the Solo battery to build things like chargers and charging pads.
 
 For information on interfacing with the Solo through the Battery Bay, visit the [Battery Bay page](hardware-batterybay.md).
 
 ## Battery Specs
 
-> <sub>* 500 grams
-
-> <sub>* Total capacity: 5.2 Ah (not a required spec)
-
-> <sub>* Charge rate: 1-2 C
-
-> <sub>* Discharge rate (continuous): 5 C
-
-> <sub>* Discharge rate (peak): 10 C
-
-> <sub>* Operating voltage range: 16.8 V to 10.0 V
-
-> <sub>* Charging temp: 0℃ ~ 40℃
-
-> <sub>* Discharging operating temp: -20℃ ~ 60℃
+* 500 grams
+* Total capacity: 5.2 Ah (not a required spec)
+* Charge rate: 1-2C
+* Discharge rate (continuous): 5C
+* Discharge rate (peak): 10C
+* Operating voltage range: 16.8 V to 10.0 V
```

Comment:
Any reason why we have range like this? - more typically ranges are low to high: 10 - to 16